### PR TITLE
Add constraint to version of leak_tracker.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: pkgs/leak_tracker_testing/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 1.0.2
+
+* Set version of leak_tracker to `any`.
```
Comment (christopherfujino) [inline pkgs/leak_tracker_testing/CHANGELOG.md:3]: update?

File: pkgs/leak_tracker_testing/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # 1.0.2
 
-* Set version of leak_tracker to `any`.
+* Update version of leak_tracker to `any`.
```
Comment (christopherfujino) [inline pkgs/leak_tracker_testing/CHANGELOG.md:3]: I mean the constraint is now ^8.0.0 and not `any`, right?

Comment (polina-c) [reply pkgs/leak_tracker_testing/CHANGELOG.md]: ha-ha, yes!

**Review by christopherfujino — APPROVED** [review]
LGTM

**Pull request merged**